README.md

```diff
@@ -1,5 +1,35 @@
-# platformsh-config-reader-python3
+# pshconfig
 
-Testing:
+[![Build Status](https://travis-ci.org/platformsh/platformsh-config-reader-python3.svg?branch=master)](https://travis-ci.org/platformsh/platformsh-config-reader-python3)
 
-    python3 -m tests.configTest
\ No newline at end of file
+A small helper to access a Platform.sh application's configuration, via [environment variables](https://docs.platform.sh/development/variables.html).
+
+Include it in your project with:
+
+```bash
+pip install pshconfig
```

Comment:
Until we do something with PyPi, this should probably be

`pip install git+https://github.com/platformsh/platformsh-config-reader-python3.git`